workflow/rules/cnvkit_panel_of_normal.smk

```diff
@@ -10,15 +6,29 @@ __license__ = "GPL3"
 
 rule cnvkit_create_targets:
     input:
-        bed=config["reference"]["design_bedfile"],
+        bed=config.get("reference", {}).get("design_bed", "")
```

Comment:
```suggestion
        bed=config.get("reference", {}).get("design_bed", ""),
```

Reply:
Could remove the design_bedfile line in the config.yaml for the integration test.